lib/vmdb/loggers/ibmcloud_logger.rb

```diff
@@ -0,0 +1,24 @@
+module Vmdb::Loggers
+  class IbmCloudLogger < VMDBLogger
```

Comment:
Ideally we could share this with the AzureLogger, there's nothing azure specific about that logger it really just filters tokens.

@Fryguy what about `ProviderSdkLogger` ?

Reply:
👍 for this... eventually we probably should just push down token filtering to the base logger, because I can't see why we wouldn't want this kind of filtering for every log message.  For now, though, rename the existing azure_logger and reuse for both Azure and IbmCloud feels like the better direction.

Reply:
changed AzureLogger to generic ProviderSdkLogger